### PR TITLE
BNH-71 bug: Fix up bugs for postcode autocomplete.

### PR DESCRIPTION
**Pull request by Chestnut-lol**

### CODE REVIEW

**Review by MaxTheComputerer — APPROVED** [review]
Looks good to me!

**Comment by Chestnut-lol** [comment]
I fixed up all bugs raised during testing the previous ticket (sending emails). All things should (hopefully) be working fine now.

One problem is that the API request (i.e. autofilling the postcode) seems to be taking 5 to 6 seconds. It was faster before I did the bugfix, but I didn't change anything with the api request so I don't know if it's just a temporary internet connection problem, or if I accidentally changed something to make it slower...

File: web/Controllers/PropertyController.cs [hunk 1]
```diff
@@ -57,36 +58,46 @@ public async Task<IActionResult> AddNewProperty_Continue([FromRoute] int step, [
...
+        {
+            if (newPropertyModel.Address.AddressLine1 == null || newPropertyModel.Address.Postcode == null)
+            {
+                // Address line 1 and postcode are the minimum information we need to create a new record
+                return View("AddNewProperty",
+                    new AddNewPropertyViewModel { Step = step, Property = newPropertyModel });
+            }
+
+            PropertyViewModel resultModel = await _azureMapsApiService.AutofillAddress(newPropertyModel);
+            newPropertyModel = resultModel;
```
Comment (BenRamchandani) [inline web/Controllers/PropertyController.cs:78]: The method mutates it anyway, so just this should work

```suggestion
            await _azureMapsApiService.AutofillAddress(newPropertyModel);
```

Might be worth changing `AutofillAddress` to just return a `Task` to make that clear

**Review by BenRamchandani — COMMENTED** [review]
One tiny thing, but this looks good

**Review by BenRamchandani — APPROVED** [review]

**Pull request merged**